frontend/mobilityai_react/App/Screens/PatientData.js

```diff
@@ -55,44 +65,68 @@ export default class PatientData extends Component {
 
     pieColour(i) {
 
-    }
+    };
 
+    componentDidMount(){
+        //TODO: Remove hard coded dates/times once date picker works
+        // let startTime = new Date(Date.UTC(2018, 11, 11, 0, 0, 0, 0)).getTime();
+        // let endTime = new Date(Date.UTC(2018, 11, 11, 23, 0, 0, 0)).getTime();
 
+        let startTime = new Date(Date.UTC(2019, 1, 9, 0, 0, 0, 0)).getTime();
```

Comment:
Use the date from the `GetDate` component?

Reply:
We'll change this once the server side issues are resolved.